docker-compose.yml

```diff
@@ -1,11 +1,16 @@
-web:
-  restart: always
-  build: ./web/
-  expose:
-    - "8000"
-  links:
-    - postgres:postgres
-  env_file: env
-  volumes:
-    - ./web:/data/web
-  command: /usr/bin/gunicorn mydjango.wsgi:application -w 2 -b :8000
\ No newline at end of file
+version: '3'
```

Comment:
Probably want to make this the most current version (3.5 ?), unless there's a compelling reason to leave it at 3.